src/effects/native/balanceeffect.cpp

```diff
@@ -55,13 +59,48 @@ EffectManifest BalanceEffect::getManifest() {
     midSide->setMaximum(1.0);
     midSide->setDefault(0.5);
 
+    EffectManifestParameter* midLowPass = manifest.addParameter();
+    midLowPass->setId("bypassFreq");
+    midLowPass->setName(QObject::tr("Bypass Freq."));
+    midLowPass->setDescription(QObject::tr("Corner frequency of the Mid Bypass LPF"));
+    midLowPass->setDescription(QObject::tr("Corner frequency ratio of the low pass filter"));
```

Comment:
The description is set twice and neither description is adequate. How about:
"Frequencies below this cutoff are not adjusted in the stereo field"